docs/usage.rst

```diff
@@ -87,6 +87,30 @@ datasets, you can use **--output-value** command-line option:
     $ cernopendata-client get-metadata --recid 1 --output-value system_details.global_tag
     FT_R_42_V10A::All
 
+If you would like to extract parts of the metadata by filtering some
```

Comment:
You can rather say:

- If the output field produces a list of values, you may want to filter only certain field values of interest. For example, you may want to ask not to output all authors of a dataset record, but only the authors matching certain particular author name. You can use the `--filter` command-line option to achieve this: